package.json

```diff
@@ -53,6 +59,16 @@
 		"tsd-check": "^0.2.1",
 		"xo": "^0.23.0"
 	},
+	"babel": {
+		"presets": [
+			[
+				"@babel/preset-env",
+				{
+					"loose": true
```

Comment:
Lets set `modules` to `umd` so it can even work without using a build-system.

Reply:
You also need to ensure `ky` is exported as a global with this config. I don't know how you do that in Babel.